x/poe/bootstrap.go

```diff
@@ -342,6 +381,34 @@ func newOCGovProposalsInitMsg(gs types.GenesisState, ocContract, engagementContr
 	}
 }
 
+// build instantiate message for the trusted circle contract that contains the arbiter pool
+func newAPTrustedCircleInitMsg(gs types.GenesisState) contract.TrustedCircleInitMsg {
+	cfg := gs.ArbiterPoolContractConfig
+	return contract.TrustedCircleInitMsg{
+		Name:                      cfg.Name,
+		EscrowAmount:              cfg.EscrowAmount.Amount,
+		VotingPeriod:              cfg.VotingRules.VotingPeriod,
+		Quorum:                    *contract.DecimalFromPercentage(cfg.VotingRules.Quorum),
+		Threshold:                 *contract.DecimalFromPercentage(cfg.VotingRules.Threshold),
+		AllowEndEarly:             cfg.VotingRules.AllowEndEarly,
+		InitialMembers:            []string{}, // sender is added to AP by default in the contract
+		DenyList:                  cfg.DenyListContractAddress,
+		EditTrustedCircleDisabled: true,
```

Comment:
This flag is copied from OC setup as described in the issue. Just want to double check with @ethanfrey if this is applies here, too.

Reply:
This is correct we do not want the AP to be able to edit the TC.